web/modules/custom/joinup_user/joinup_user.post_update.php

```diff
@@ -53,3 +45,20 @@ function joinup_user_post_update_remove_obsolete_og_roles_changed_message_config
 function joinup_user_post_update_add_never_autoplay_permission() {
   user_role_grant_permissions(AccountInterface::AUTHENTICATED_ROLE, ['never autoplay videos']);
 }
+
+/**
+ * Remove the permissions to manage subscriptions.
+ */
+function joinup_user_remove_subscription_permissions() {
+  // Separate permissions are not needed for this since the subscription
+  // settings are part of the user profile. It is fully covered by the
+  // 'administer users' permission, and users are always allowed to edit their
+  // own profiles.
+  /** @var \Drupal\user\RoleInterface $role */
+  foreach (Role::loadMultiple() as $role) {
+    foreach (['manage own permissions', 'manage all permissions'] as $permission) {
```

Comment:
`manage own permissions`, `manage all permissions`? There are not such permissions in Joinup. If you've misspelled `manage own subscriptions` and `manage all subscriptions` then we still don't need this update script. No roles are granted right now with these permissions and few lines above you deleted the script that adds such permissions.

Reply:
Yes this should be `manage own subscriptions` and `manage all subscriptions`.

These permissions are in fact used in Joinup. They were added a long time ago in ISAICP-2243 when we first started the work on the subscriptions. But these permissions don't add any value so let's get rid of them.